docs/install.md

```diff
@@ -55,9 +56,16 @@ $ kubectl gadget version
 ```
 
 ### Compile from source
+#### Build requirements
 
-To build Inspektor Gadget from source, you'll need to have a Golang version
-1.19 or higher installed.
+To build Inspektor Gadget from source, the following requirements need to be installed and configured on your system:
+- Golang version 1.19 or higher
+- Docker Engine
+- gcc compiler
+- make
```

Comment:
Did you also need the following?

- pkg-config
- libseccomp-dev

Reply:
Yees, missed these ones. Added now. Thanks for checking.